public/launch-context.json

```diff
@@ -1,5 +1,5 @@
 {
-  "clientId": "6c12dff4-24e7-4475-a742-b08972c4ea27",
-  "scope": "patient/*.read launch/patient",
+  "clientId": "a0f32a02-c124-4ba7-858b-5a52b9591de6",
```

Comment:
@mcjustin you may need to edit this value (in the same PR) to match what Epic gives us

Reply:
@ivan-c  done now